core/workspace.js

```diff
@@ -110,6 +110,13 @@ Blockly.Workspace = function(opt_options) {
  */
 Blockly.Workspace.prototype.rendered = false;
 
+/**
```

Comment:
this also is the diff from another PR (which exists in my "performance" base branch, please ignore for now)